sigstore/_internal/merkle.py

```diff
@@ -91,10 +91,11 @@ def _hash_leaf(leaf: bytes) -> bytes:
     return hashlib.sha256(data).digest()
 
 
-def verify_merkle_inclusion(
-    inclusion_proof: RekorInclusionProof, entry: RekorEntry
-) -> None:
+def verify_merkle_inclusion(entry: RekorEntry) -> None:
```

Comment:
Not related to this PR, but something we need to add - We now return checkpoints (https://github.com/sigstore/rekor/blob/main/openapi.yaml#L608), signed root hashes. We need to add verification of these when verifying the inclusion proof, otherwise we're just verifying a list of hashes without any connection to the log.

You'll need to verify 1) the signature on the checkpoint and 2) the root hash in the checkpoint matches the root hash from the inclusion proof.

There's a code example here - https://github.com/sigstore/rekor/blob/4b1fa6661cc6dfbc844b4c6ed9b1f44e7c5ae1c0/pkg/verify/verify.go#L114

Reply:
cc @asraa

Reply:
This is only for online rekor entries! But yes, I was going to file an issue on this after reading this.

Reply:
Please do, thanks!

Reply:
#248.